src/StartMenu.ts

```diff
@@ -296,64 +299,86 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const plugin = getPlugin(EXTENSION_POINT_ID, EXTENSION_ID);
```

Comment:
otherwise I would have to loop over all extensions and check if they have a headers property, wouldn' I?
what if other extensions with the same `EXTENSION_POINT_ID` also have headers?

I was not sure if this is the right way to get the meta data I need at this point. What is the correct approach instead?